install_scripts/install_sindri.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+# Install sindri and related software
+
+if [[ $# -ne 1 ]]; then
+    echo "Pass the sensor number"
+    exit 1
+fi
+
+name="hamma"$1  # This will be used for the server push location
+
+# We need to modify a file (for now). Set up the variables for that
+LEKTOR_FILE="/home/pi/dev/sindri/src/sindri/website/mjolnir-website/mjolnir-website.lektorproject"
+LEKTOR_KEY="target = rsync://pi@hamma.dev/var/www/hamma.dev/public_html/"
+
+INSTALL_PATH="/home/pi/dev/"  # Trailing slash, please
+VENV_NAME="sindrienv"
+
+
+# Now, we're going to set up the environment
+
+python3 -m venv $INSTALL_PATH$VENV_NAME
+
+cp $INSTALL_PATH$VENV_NAME"/bin/activate" "/home/pi/$VENV_NAME"
+
+source "/home/pi/$VENV_NAME"
+
+# Note: not sure if this is needed
+pip install --upgrade setuptools wheel
```

Comment:
If its a new env, then yes. Though this is missing `pip`, so it might get an ancient version...

```suggestion
pip install --upgrade pip setuptools wheel
```